site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,478 @@
+
+WebAuthn focuses on the browser interaction with a passwordless protocol. CTAP2 is a complementary standard which handles communication between the browser and the hardware or software authenticator. (The yellow smiley face below is the user and the white globe is the browser.)
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/ctap2-webauthn.png" alt="Showing the difference between CTAP2 and WebAuthn." class="img-fluid" figure=false %}
+
+If you have a website that lets a user log in with an authenticator, then CTAP2 is handling communication between the authenticator and the browser. 
+
+You can use CTAP2 without WebAuthn, for example, with a native app.
+
+## Selling Clownwear
+
+I promised clowns, did I not? Assume you have a website that you want to use to sell clownwear. You know, big shoes and red noses and the like? You will call it... Cosmo's Clown Store.
+
+{% include _image.liquid src="/assets/img/advice/webauthn-explained/clownwear.png" alt="Clownwear examples." class="img-fluid" figure=false %}
+
+In order to use our store, people need to log in to perform tasks like checking their order status, adding things to their cart, and placing orders. Logging in will tie the user's identity to other data like their mailing address and credit card. If you want to let folks log in securely, what are your options?
+
+### Passwords
+
+Passwords are a solid choice, but can be forgotten, stolen or re-used. In fact, over 11B username/password pairs are publicly available on the site HaveIBeenPwned. User's aren't really fans of remembering passwords either, and insecurely re-use them across different systems.
+
+This means that user accounts on your system don't simply depend on your security posture, but on the posture of all the other sites that the user has registered for with the same password.
+
+Doh!
+
+### Password Managers
+
+Well, what about password managers?
+
+The user experience can be tough when dealing with password managers. If you're using a password manager that integrates into the browser, any non-standard login form UX can be difficult for it to handle.
+
+If a password manager is centralized, it is now a high-value target. If it is not, then users can't login to accounts from more than one device.
+
+### MFA
+
+If you add an additional factor of authentication (i.e. text the user a code, send them an email with a code, or require them to enter some code based on some algorithm using something like Google Authenticator), you can increase security. These additional factors of the authentication.
+
+They add additional assurance about who the user is during an authentication process. However, the user experience is tough. It can be a hassle to force someone to go to a different device to get a code. Depending on the method, it can also be insecure. For example, SMS codes can be phished.
```

Comment:
It's not the topic of this article, but the recent trend of bombarding people with push messages for MFA login until they get tired of it and just accept is troubling as well.